lib/rubocop/cop/lint/multiple_comparison.rb

```diff
@@ -25,9 +25,11 @@ class MultipleComparison < Base
         extend AutoCorrector
 
         MSG = 'Use the `&&` operator to compare multiple values.'
+        COMPARISON_METHODS = %i[< > <= >=].freeze
+        RESTRICT_ON_SEND = COMPARISON_METHODS
 
         def_node_matcher :multiple_compare?, <<~PATTERN
-          (send (send _ {:< :> :<= :>=} $_) {:< :> :<= :>=} _)
+          (send (send _ {:< :> :<= :>=} $_) {:#{COMPARISON_METHODS.join(' :')}} _)
```

Comment:
FYI, if `COMPARISON_METHODS` was a set, you could use `%COMPARISON_METHODS` here... but I'm also working on optimizing `{:list :of :literals}` into a set, so no need to change this. For the future though, might be more readable :-)